ci-automation/release.sh

```diff
@@ -93,7 +93,11 @@ function _inside_mantle() {
     secret_to_file google_release_credentials_file "${GOOGLE_RELEASE_CREDENTIALS}"
 
     for platform in aws azure; do
-      for arch in amd64 arm64; do
+      arches=(amd64 arm64)
+      if [[ "${CHANNEL}" == "lts" ]] && [[ "${platform}" == "azure" ]] && echo "$VERSION" | grep -q "^3033"; then
+        arches=(amd64)
+      fi
+      for arch in "${arches[@]}"; do
```

Comment:
Next time someone could want to use `${arches[@]}` in the plume release for loop and this global change prevents the run of plume release for arm64. I would rather call it `arches_prerelease`.

```suggestion
      arches_prerelease=(amd64 arm64)
      if [[ "${CHANNEL}" == "lts" ]] && [[ "${platform}" == "azure" ]] && echo "$VERSION" | grep -q "^3033"; then
        arches_prerelease=(amd64)
      fi
      for arch in "${arches_prerelease[@]}"; do
```